.gitignore

```diff
@@ -62,6 +63,7 @@ src/ontology/imports/*_terms_combined.txt
 src/ontology/components/*.owl
 src/ontology/components/*.json
 src/ontology/components/*_terms_combined.txt
+src/ontology/components/*relation-graph*
```

Comment:
@matentzn Should we just ignore all of `src/ontology/components/*`?

Reply:
makes sense